$NetBSD: patch-CMakeLists.txt,v 1.2 2016/02/03 19:34:33 bouyer Exp $

Don't use ${CMAKE_MODULE_PATH} directly as it will be changed
and containt multiple directories

--- CMakeLists.txt.orig	2015-12-14 20:13:21.000000000 +0100
+++ CMakeLists.txt	2015-12-14 20:15:13.000000000 +0100
@@ -826,7 +826,7 @@
 # "make uninstall" rules
 #================================================
 configure_file(
-    "${CMAKE_MODULE_PATH}/cmake_uninstall.cmake.in"
+    "${PROJECT_SOURCE_DIR}/CMakeModules/cmake_uninstall.cmake.in"
     "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
     IMMEDIATE @ONLY )
 
